config/manifests/patches/csvAnnotations.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: ibm-websphere-liberty.v0.0.0
   namespace: placeholder
   annotations:
-    containerImage: IMAGE
-    createdAt: CREATEDAT
\ No newline at end of file
+    containerImage: default-route-openshift-image-registry.apps.hyperion-cnb.cp.fyre.ibm.com/websphere-liberty/operator:latest
```

Comment:
not sure why git is checking these changes in, I thought it was already added to to ignore list, maybe not